src/DurableTask.Netherite/StorageLayer/Faster/StoreWorker.cs

```diff
@@ -573,12 +581,6 @@ protected override async Task Process(IList<PartitionEvent> batch)
                     markPartitionAsActive = markPartitionAsActive || partitionEvent.CountsAsPartitionActivity;
                 }
 
-                // if we are processing events that count as activity, our latency category is at least "low"
-                if (markPartitionAsActive)
-                {
-                    this.loadInfo.MarkActive();
-                }
-
```

Comment:
after this deleted piece of code, this method may return if it is shutting down. Could we be losing information by no longer recording the latency right before a shut down?

Reply:
I think that would be o.k. since once we shut down we are no longer reporting the partition load anyway. The partition will be started somewhere else and start reporting load from there.